seatunnel-dist/src/main/docker/Dockerfile

```diff
@@ -8,7 +8,6 @@ RUN cd /opt && \
     tar -zxvf apache-seatunnel-${VERSION}-bin.tar.gz && \
     mv apache-seatunnel-${VERSION} seatunnel && \
     rm apache-seatunnel-${VERSION}-bin.tar.gz && \
-    cp seatunnel/config/log4j2_client.properties seatunnel/config/log4j2.properties && \
```

Comment:
The config are different now, so skip copy this file.

Copy this file is want to change the log output to console instead of of File. Then when deploy on docker, we can use `docker logs` to check the log, don't need check the file in container.

Reply:
I prefer old way. It is more compatible and easier to integrate.

Reply:
updated
update config in docker file